### DISCUSSION TITLE
Where I can find prebuilt binaries?

### DISCUSSION THREAD

**Discussion by Akiyamka** [romgrk/node-gtk#367] [Q&A] [answered]
Readme says that 

> Note that prebuilt binaries are available for common systems, in those cases building is not necessary.

and

> Linux: prebuilt binaries available

but I can't find any links to it

**Comment by romgrk** [comment]
The package is setup to download prebuilt binaries when you `npm install` it.

**Reply by Akiyamka** [reply]
Looks like it can't found binaries for my system. Where i can find full list of available binaries?

**Reply by romgrk** [reply, marked as answer]
Not publically available, it's in an S3:
![image](https://github.com/romgrk/node-gtk/assets/1423607/40de4c62-a179-4e1e-ad29-84e9f2bc6661)

I haven't had much time to put in the project lately, the macOS binaries haven't been building for a while.

**Reply by matthewruzzi** [reply]
@romgrk  Can you host the binaries on GitHub and build them with GitHub Actions?

**Reply by romgrk** [reply]
It would be possible I think, but the build & install scripts would need to be updated.

**Reply by matthewruzzi** [reply]
If they are generated and hosted on GitHub they can be verified with artifact attestations. 
https://docs.github.com/en/actions/security-for-github-actions/using-artifact-attestations/using-artifact-attestations-to-establish-provenance-for-builds
